src/index.html

```diff
@@ -7,8 +7,47 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    >
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    >
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a class="logo-link">
+        <img
+        src="./images/logo.png"
+        alt="Logo des Geschäftes"
+        class="logo"
+        >
+      </a>
+      <nav class="nav_list">
+        <ul class="ul_list">
+          <li class="lists_part"><a href = "#" class="link_in_nav is-active">Apple</a></li>
```

Comment:
```suggestion
          <li class="lists_part">
            <a href = "#" class="link_in_nav is-active">Apple</a>
          </li>
```
fix formatting everywhere